tests/framework/CodeInterfaceTests/CobraTF/gold/CobraTFinterfaceTest2/testDummyStep_Grid/2/small_3x3rod_HFP.inp

```diff
@@ -0,0 +1,443 @@
+***********************************************************************************************
```

Comment:
are we OK with putting this input file in the repo?
are we OK from an export control point of view?

Reply:
I checked it with NCSU team (Paolo Balestra, Maria Avramova) and there's no export control issue.